WalletWasabi.Gui/Controls/WalletExplorer/TransactionBroadcasterViewModel.cs

```diff
@@ -82,6 +83,20 @@ public TransactionBroadcasterViewModel(WalletViewModel walletViewModel) : base("
 						Title = "Import Transaction"
 					};
 
+					if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+					{
+						var initailDirectory = Path.Combine("/media", Environment.UserName);;
```

Comment:
nit: 2 semicolons

Reply:
Also rename `initailDirectory` to `initialDirectory`